### ISSUE TITLE
GUI: Output all reports as combined export?

### ISSUE THREAD

**Issue by kgrons** [MediaArea/MediaConch#175]
In the GUI, is there a way to export a summary of all per-file reports (equivalent of using `--Format=xml -fa` in the CLI version)?

Docs in _Help_ > _How to Use_ > _Exporting Reports_ only refer to downloading individual file reports.

**Comment by kellyhaydon** [comment]
+1

**Comment by JeromeMartinez** [comment]
No, we expected that people use CLI for all files and GUI for a per file export.
But this is something we could add.
